contrib/terraform/aws/.gitignore

```diff
@@ -1,2 +1,2 @@
 *.tfstate*
-inventory
```

Comment:
This was because in the old version the inventory file was generated at the Terraform project directory. In the new version I put the inventory file into the right location for ansible to  save the -i flag